app/helpers/summary_cards_helper.rb

```diff
@@ -1,10 +1,10 @@
 module SummaryCardsHelper
   def summary_card_for(resource)
     {
-      Group => 'shared/summary_cards/group',
-      GroupsUser => 'shared/summary_cards/member',
-      List => 'shared/summary_cards/list',
-      Resource => 'shared/summary_cards/resource',
-    }[resource.class]
+      'Group' => 'shared/summary_cards/group',
```

Comment:
Weirdest thing ever. It wasn't working with symbols for the `Resource` class so I decided to switch to strings. I don't know why it happened (only for `Resource`) and didn't want to spend too much time trying to figure it out. I did some checking of course, and was getting the correct `Resource` class (being passed and in the hash) but for some reason they did not match.